experiments/integrated/ozark/allocations/ozark_allocations.jl

```diff
@@ -0,0 +1,128 @@
+import SciMLBase
+import ClimaTimeSteppers as CTS
+using ClimaCore
+using Plots
+using Statistics
+using Dates
+
+using ClimaLand
+using ClimaLand.Domains: Column
+using ClimaLand.Soil
+using ClimaLand.Soil.Biogeochemistry
+using ClimaLand.Canopy
+using ClimaLand.Canopy.PlantHydraulics
+import ClimaLand
+import ClimaLand.Parameters as LP
+import ClimaParams
+
+PROFILING = false
+try
+    import Profile, ProfileCanvas
+    global PROFILING = true
+    @info "ProfileCanvas found, running with profiler"
+catch
+end
+global FT = Float64
+climaland_dir = pkgdir(ClimaLand)
+global site_ID = "US-MOz"
+savedir = joinpath(climaland_dir, "experiments/integrated/ozark/allocations")
+global earth_param_set = LP.LandParameters(FT)
+
+# Create model and setup some timestepping parameters
+include(
```

Comment:
duplicate code is in the simulation setup and in the initial conditions. Note that we should reset initial conditions in between runs, in particular `p` may be updated in place during the sim. so that is why that is in a separate file from the rest of the model setup.

Ideally these would not be `include` statements, but instead be calling functions. but both included scripts require global variables and I had a world age issue when I tried it. Im not sure. maybe there is an easy fix